components/azure-iot/sample/main/azure_main.c

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
As a general rule, example source files are released into public domain or CC0 licensed. As in:

https://github.com/espressif/esp-idf/blob/a5eb369126e5cef31715d2755931cf933bb98bee/examples/get-started/hello_world/main/hello_world_main.c#L1-L8 


Please check if this example license can be updated accordingly.